CHANGELOG.md

```diff
@@ -403,6 +403,7 @@ Compatibility release so users can upgrade RuboCop to 0.51.0. No new features.
 * Add `RSpecMultipleDescribes` cop to check for multiple top level describes. ([@geniou][])
 * Add `RSpecDescribedClass` to promote the use of `described_class`. ([@geniou][])
 * Add `RSpecInstanceVariable` cop to check for the usage of instance variables. ([@geniou][])
+* Add `FactoryBot/FactoryClassName` cop. ([@jfragoulis][])
```

Comment:
Ah, just one more thing. Could you move this line up towards the top of the file, as the last bullet point under `## Master (Unreleased)` please.

Reply:
…you‘ll need to rebase on latest master too, to avoid merge conflict.

Reply:
Wow, what an oversight! :) :) Done and thanks!